lib/terraforming/resource/redshift.rb

```diff
@@ -0,0 +1,64 @@
+module Terraforming
+  module Resource
+    class Redshift
+      include Terraforming::Util
+
+      def self.tf(client: Aws::Redshift::Client.new)
+        self.new(client).tf
+      end
+
+      def self.tfstate(client: Aws::Redshift::Client.new)
+        self.new(client).tfstate
+      end
+
+      def initialize(client)
+        @client = client
+      end
+
+      def tf
+        apply_template(@client, "tf/redshift")
+      end
+
+      def tfstate
+        clusters.inject({}) do |resources, cluster|
+          attributes = {
```

Comment:
`cluster_identifier` should be included in tfstate.
This causes re-create Redshift cluster by `terraform apply`:

```
-/+ aws_redshift_cluster.dtan4-hogefuga
    allow_version_upgrade:               "true" => "1"
    automated_snapshot_retention_period: "1" => "1"
    availability_zone:                   "ap-northeast-1a" => "ap-northeast-1a"
    cluster_identifier:                  "" => "dtan4-hogefuga" (forces new resource)
```